packages/universal-components/src/Hoverable/HoverMonitor.js

```diff
@@ -1,32 +0,0 @@
-// @flow
-
-const HoverMonitor = () => {
```

Comment:
I think we need to include something similar to this into new `withHover`.
To prevent components being stuck in hover mode in web version on mobile/touch devices.
What we need is to basically disable hover events dispatching when there is touch detected.

It happens for example on current result cards.

![hover](https://user-images.githubusercontent.com/2660330/57069138-07214c00-6cd4-11e9-93f9-ba15bb97c855.gif)

Reply:
Oops sure! Added it back 💃

Reply:
Thanks, now it works fine! 🚀